### PR TITLE
Sending email with Emailjs 

### PR DESCRIPTION
**Pull request by Souvikns**
Created a helper function to send emails to any email id provided by the user. 
#5  working on  
- [x] basic email message

### CODE REVIEW

File: src/lib/email.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            email: email
+        }
+    ).then(res => {
+        return { successfull: true, res, message: 'Email successfully created' }
+    }).catch(err => {
+        return { successfull: false, err, message: 'Some error occured' }
+    })
+}
+
+export { sendEmail }
```
Comment (abhu-A-J) [inline src/lib/email.js:25]: Make the send Email a default export! Rest of it LGTM!

Comment (Souvikns) [reply src/lib/email.js:25]: ok

**Comment by Souvikns** [comment]
I have made sendEmail default export

**Comment by abhu-A-J** [comment]
> I have made sendEmail default export

Try making PR from a separate branch from next time instead of the master! This looks good to me , I'm merging it! Good job!

**Comment by abhu-A-J** [comment]
@anushbhatia You may assign the correct points label!

**Pull request merged**